src/docs/deployment/web-renderers.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Web renderers
+description: How to choose a web renderer for running and building a web app.
+---
+
+Flutter provides two ways to display an application on the web:
+
+* **HTML**: Uses a combination of HTML elements, CSS, Canvas elements, and SVG
+  elements. This renderer has a smaller download size, 200KB for a simple app.
+* **CanvasKit:** Uses Skia compiled to WebAssembly and rendered to WebGL. This
+  renderer is fully consistent with Flutter mobile and desktop, has faster
+  performance, and is less likely to have differences across browsers, but adds
+  about 2MB in download size.
+
+## Command line flags
+
+The `--web-renderer` command line option takes one of three values, `html`,
+`canvaskit`, or `auto`.
+
+Choose the **html** renderer when optimizing for code size over performance on
+desktop and mobile devices.
```

Comment:
I find this confusing. Maybe we can chat?

Reply:
agreed a chat would be good to go over this

Reply:
There's more information in [this design doc](https://docs.google.com/document/d/1aY0iU16wf_sdT7nwfpjgT-IatHNfF3slTiYHKmxcIog) that perhaps we can integrate here.

Reply:
What do. you mean "on desktop and mobile devices"? Choose this for an Android/iOS app? How does it even affect that? Choose when running either a desktop app or a WEB app on mobile, like Chrome? This is unclear.

Reply:
I see, I'm trying to differentiate from `auto` here. You should choose the `html` renderer if you care about code size **regardless of whether or not the app is accessed from a mobile browser or desktop browser**. 

The `auto` option should be chosen if you care more about code size when the app is accessed from a mobile browser, and if you care about fast performance when the app is accessed from a desktop browser. 

The `auto` option is provided to handle what we think is the most common use case; that most apps will choose to use HTML for their users who are using a mobile browser (e.g. desktop Safari) and would prefer that their users see the CanvasKit version for their users who are using a desktop browser.